tests/cloudlaunchprocess/test.csv

```diff
@@ -0,0 +1,25 @@
+action,ms,bytes,lang,code,repeat,minver,comment
+comment,0,,"Test 1 - Launch and then kill a process with all possible cmd line args"
+run,0,0,q,.sys.launch input1,,,"Start a new process with extra command line args"
+run,0,0,q,system "sleep 2",,,"Wait for process to start"
+run,0,0,q,h:hopen `::7124:admin:admin,,,"Attempt to connect to the new process"
+true,0,0,q,`test2=@[h;".proc.procname";`],,,"Check that connection has been made"
+true,0,0,q,11011111010b~(value input1) in' value key[input1]#.Q.opt (h)".z.X",,,"Tentatively? - Check that the cmd line args are as expected"
+run,0,0,q,pid1:h".z.i",,,"Find the PID of the launched process"
+run,0,0,q,.sys.kill "test2",,,"Kill the process"
+true,0,0,q,not h in key[.z.W],,,"Check that the handle to the process is no longer open"
+true,0,0,q,not pid1 in "I"$1 _ .proc.sys "ps -u $USER | awk '{print $1}'",,,"Check that the PID of the old process is no longer in use"
+
+comment,0,,"Test 2 - Launch and then kill a process with no extra cmd line args, to test defaulting"
+run,0,0,q,.sys.launch input2,,,"Start a new process with no extra command line args, hence relying on defaulting in cloudutils.q"
+run,0,0,q,system "sleep 2",,,"Wait for process to start"
+run,0,0,q,proc_port:first exec hpup from .servers.querydiscovery`test where procname=`test3,,,"Find process port from discovery services "
+run,0,0,q,"h2:hopen `$string[proc_port],"":admin:admin""",,,"Attempt to connect to the process"
+true,0,0,q,`test3=@[h2;".proc.procname";`],,,"Check that connection has been made"
+true,0,0,q,110b~(value input2)in' value key[input2]#.Q.opt h2".z.X",,,"To be reworked:Check that correct process name and type are present in the cmd line args of the process"
```

Comment:
as per comment, is this still to be reworked or should comment be updated to remove that?